js/app.js

```diff
@@ -0,0 +1,83 @@
+
+//question 1
+
+let obj ={
+	insideObj: {
+		key1: 'yo';
+		key2: 12
+	}
+
+}
+
+console.log(insideObj.key1())
+
+// question 2
+
+let obj = {
+	 {
+		someObject: {
+			someProperty: 'hello', 'hi', 'how are you'
+ 		}
+	}
+ }
+
+ // question 3
+
+ let obj = {
+ 	property () => { ['dare', 'james']
+ 	}
+ }
+
+ // question 4
+
+ let fruits = ['james', 'milton', 'marcus':{height: 6, age: 23, }, 'david']
+
+ console.log(fruits[2].())
+
+ // question 5 
+
+ var arr = ['this is the first element', 'this is the second element': [1,2,3,4], 'this is the last element'];
+
+ // question 6
+
+ var years = [1950, 1960, 1970: {function myfunction = function namedFunction(){
+    statements
```

Comment:
you should use an arrow function.  and dont put it in {}.  that's for objects.  also you did not call the function as it asked.